### DISCUSSION TITLE
Does this work on an existing save

### DISCUSSION THREAD

**Discussion by CaptainRosh** [TEHE-Studios/ExpandedHelicopterEvents#265] [Q&A] [answered]
Hello. I am just wondering if this mod works on an existing save, which is my 4 (in-game) month world. I had the vanilla helicopter enabled in my first week. Can I put this in?

**Comment by Chuckleberry-Finn** [comment, marked as answer]
Yes it works, but you'll need to adjust the default sandbox setting for the mod.

**Reply by CaptainRosh** [reply]
Oh cool, what do i need to adjust then?
